<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>100</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="stud_tab">
       <attribute name="title">
        <string>Students</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="stud_table">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="autoScroll">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideRight</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="work_tab">
       <attribute name="title">
        <string>Work</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="work_table">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="res_tab">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="res_table">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logs_tab">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="1" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="logsTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>100</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="logsPathLabel">
          <property name="text">
           <string>Path To Log</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Students"/>
    <addaction name="actionLoad_Work"/>
    <addaction name="actionLoad_Results"/>
   </widget>
   <addaction name="menuTable"/>
  </widget>
  <action name="actionCompile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Compile Resulting Table</string>
   </property>
  </action>
  <action name="actionLoad_Students">
   <property name="text">
    <string>Load Students</string>
   </property>
  </action>
  <action name="actionLoad_Work">
   <property name="text">
    <string>Load Work</string>
   </property>
  </action>
  <action name="actionLoad_Results">
   <property name="text">
    <string>Load Results</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
